thing.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python3
```

Comment:
Yeah - that's basically just a command line tool for exercising the web API. I plan on rewriting it in node. When I was working on foxbox, I found that it sometimes took quite a while between when the WebAPI was ready and the web interface was able to use it, and without a command line tool, I wouldn't have been able to test anything.